features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/DeploymentOnbording.jsx

```diff
@@ -0,0 +1,312 @@
+import CardHeader from '@material-ui/core/CardHeader';
+import PropTypes from 'prop-types';
+import { useAppContext } from 'AppComponents/Shared/AppContext';
+import { useTheme } from '@material-ui/core';
+import Checkbox from '@material-ui/core/Checkbox';
+import RadioButtonUncheckedIcon from '@material-ui/icons/RadioButtonUnchecked';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+
+const useStyles = makeStyles(() => ({
+    root: {
+        minHeight: '480px',
+    },
+    textOptional: {
+        fontFamily: 'sans-serif',
+        fontSize: 'small',
+        color: '#707070',
+        fontWeight: '100',
+    },
+    textRevision: {
+        fontFamily: 'sans-serif',
+        fontSize: '16px',
+        color: '#707070',
+        fontWeight: '800',
+    },
+    textDeploy: {
+        fontFamily: 'sans-serif',
+        fontSize: '26px',
+        color: '#1B3A57',
+        fontWeight: '800',
+    },
+    textDescription: {
+        fontFamily: 'sans-serif',
+        fontSize: '16px',
+        color: '#707070',
+        fontWeight: '400',
+    },
+    descriptionWidth: {
+        minWidth: '550px',
+    },
+    textAlign: {
+        textAlign: 'center',
+    },
+}));
+/**
+ * Renders an Deployment Onboarding
+ * @class Environments
+ * @extends {React.Component}
+ */
+export default function DeploymentOnboarding(props) {
+    const {
+        classes,
+        getVhostHelperText,
+        createDeployRevision,
+        description,
+        setDescription,
+    } = props;
+    const classes1 = useStyles();
+    const theme = useTheme();
+    const { maxCommentLength } = theme.custom;
+    const { settings } = useAppContext();
+    const defaultVhosts = settings.environment.map(
+        (e) => (e.vhosts && e.vhosts.length > 0 ? { env: e.name, vhost: e.vhosts[0].host } : undefined),
+    );
+    const [DescriptionOpen, setDescriptionOpen] = useState(false);
+    const [SelectedEnvironment, setSelectedEnvironment] = useState([]);
+    // const [currentLength, setCurrentLength] = useState(0);
+    const [selectedVhostDeploy, setVhostsDeploy] = useState(defaultVhosts);
+
+    const handleDescriptionOpen = () => {
+        setDescriptionOpen(!DescriptionOpen);
+    };
+
+    const handleVhostDeploySelect = (event) => {
+        const vhosts = selectedVhostDeploy.filter((v) => v.env !== event.target.name);
+        vhosts.push({ env: event.target.name, vhost: event.target.value });
+        setVhostsDeploy(vhosts);
+    };
+
+    const handleChange = (event) => {
+        if (event.target.checked) {
+            setSelectedEnvironment([...SelectedEnvironment, event.target.value]);
+        } else {
+            setSelectedEnvironment(
+                SelectedEnvironment.filter((env) => env !== event.target.value),
+            );
+        }
+        if (event.target.name === 'description') {
+            setDescription(event.target.value);
+            // setCurrentLength(event.target.value.length);
```

Comment:
Shall we remove this commented line

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/10270/commits/ee20a3b9b54f0df2bafd12c6a3e622938b22935e